File: src/main/java/com/github/bjansen/mintellij/psi/MintElement.java
package com.github.bjansen.mintellij.psi;

import com.intellij.extapi.psi.ASTWrapperPsiElement;
import com.intellij.lang.ASTNode;
import org.jetbrains.annotations.NotNull;

public class MintElement extends ASTWrapperPsiElement {

	public MintElement(@NotNull ASTNode node) {
		super(node);
	}
}

File: src/main/kotlin/com/github/bjansen/mintellij/psi/MintElement.kt
package com.github.bjansen.mintellij.psi

import com.intellij.extapi.psi.ASTWrapperPsiElement
import com.intellij.lang.ASTNode
import com.intellij.psi.NavigatablePsiElement

open class MintElement(node: ASTNode) : ASTWrapperPsiElement(node), MintPsiElement

interface MintPsiElement : NavigatablePsiElement
